### ISSUE TITLE
Undefined variable error

### ISSUE THREAD

**Issue by NicktheGeek** [equalizedigital/accessibility-checker#82]
The first time a post is edited there can be an undefined variable error. 
```
Warning: Undefined variable $found_value in .../accessibility-checker/includes/rules/color_contrast_failure.php on line 732
```

<img width="1512" alt="Screen Shot 2022-12-02 at 12 53 07 PM" src="https://user-images.githubusercontent.com/835568/205356642-603c5b42-3370-44a9-b1c9-0620fdf9a333.png">

**Comment by SteveJonesDev** [comment]
Added to version 1.3.9
